draft-ietf-httpbis-resumable-upload.md

```diff
@@ -470,27 +470,16 @@ Specification: This document
 
 * Split the Upload Transfer Procedure into the Upload Creation Procedure and the Upload Appending Procedure.
 
-# Acknowledgments
-{:numbered="false"}
-
-This document is based on an Internet-Draft specification written by Jiten Mehta, Stefan Matsson, and the authors of this document.
-
-TODO: more acknowledgements.
 
-# Appendix
```

Comment:
If I see it correctly, there is no Appendix heading in the document now. Is this intended?

Reply:
The title of appendicies is handled automatically by the tooling. Anything after the `-- back` separator will be an appendix unless otherwise stated via a decoration.

This is clearer in the preview render at https://httpwg.org/http-extensions/lucas/resumable-uploads-appendix/draft-ietf-httpbis-resumable-upload.html where the sections are more clearly labelled as Appendix A, B, C etc

Reply:
Great, thank you for the explanation.